File: WS/Models/IN/CommandesSearchParameters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WS.Models.IN
{
    public class CommandesSearchParameters
    {

        public Int32? Id { set; get; }
        public String DtMin { set; get; }
        public String DtMax { set; get; }
        public Int32? ProduitId { set; get; }
        public String ProduitReference { set; get; }
        public Int32? EleveId { set; get; }
        public String ReferenceTransaction { set; get; }
        public String ReferenceExterne { set; get; }
        public Int32? StatutId { set; get; }

    }

}

File: Angular/app/Commandes/Models/CommandesSearchParameters.ts
export class CommandesSearchParameters {


    constructor() {

        this.Id = null;

        this.DtMin = null;
        this.DtMax = null;

        this.ProduitId = null;
        this.ProduitReference = null;

        this.EleveId = null;

        this.ReferenceTransaction = null;
        this.ReferenceExterne = null;

        this.StatutId = null;

    }

    public Id: number | null;

    public DtMin: string | null;
    public DtMax: string | null;

    public ProduitId: number | null;
    public ProduitReference: string | null;

    public EleveId: number | null;

    public ReferenceTransaction: string | null;
    public ReferenceExterne: string | null;

    public StatutId: number | null;


}